### PR TITLE
remove old 6.11 warnings from Windows Agent docs

### PR DESCRIPTION
**Pull request by shanelhuang**
### What does this PR do? What is the motivation?
Removes a lot of the warnings for really old versions of the Windows Agent that don't apply anymore, and other update parts of the Windows Agent usage pages. 

https://datadoghq.atlassian.net/browse/WINA-198

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by alai97 — DISMISSED** [review]

File: content/en/agent/guide/windows-agent-ddagent-user.md [hunk 1]
```diff
@@ -19,15 +16,13 @@ The account is assigned to the following groups during installation:
...
+Additionally, the following security policies are applied to the account during installation:
 * Deny access to this computer from the network
 * Deny log on locally
 * Deny log on through Remote Desktop Services
 * Log on as a service
 
-**Important**: Since the account is modified during installation to restrict its privileges, including login privileges, make sure it is not a 'real' user account but an account solely dedicated to run the Datadog Agent.
-
-**Note**: All example commands on this page use `<>` to indicate a variable that should be replaced. For example, if the user account is `ddagentuser` and the command contains `DDAGENTUSER_NAME=<USERNAME>` then `DDAGENTUSER_NAME=ddagentuser` should be entered in the command line.
+The Windows Agent can also use a user-supplied account. Since the account is modified during installation to restrict its privileges, including login privileges, make sure it is not a 'real' user account but an account solely dedicated to run the Datadog Agent.
```
Comment (hestonhoffman) [inline content/en/agent/guide/windows-agent-ddagent-user.md:25]:
```suggestion
The Windows Agent can also use a user-supplied account. Do not use a 'real' user account. The user-supplied account should be solely dedicated to running the Datadog Agent. The account is modified during installation to restrict its privileges, including login privileges.
```

File: content/en/agent/guide/windows-agent-ddagent-user.md [hunk 2]
```diff
@@ -37,15 +32,15 @@ If no user account is specified on the command line, the installer attempts to c
 
 If a user account is specified on the command line, but this user account is not found on the system, the installer attempts to create it. If a password was specified, the installer uses that password, otherwise it generates a random password.
 
-To specify the optional USERNAME and PASSWORD on the command line pass the following properties to the `msiexec` command (Remove the bracket `<>` characters from the username and password placeholders):
+To specify the optional USERNAME and PASSWORD on the command line pass the following properties to the `msiexec` command (The bracket `<>` characters indicate a variable that should be replaced):
```
Comment (hestonhoffman) [inline content/en/agent/guide/windows-agent-ddagent-user.md:35]:
```suggestion
To specify the optional USERNAME and PASSWORD on the command line, pass the following properties to the `msiexec` command (The bracket `<>` characters indicate a variable that should be replaced):
```

File: content/en/agent/guide/windows-agent-ddagent-user.md [hunk 3]
```diff
@@ -37,15 +32,15 @@ If no user account is specified on the command line, the installer attempts to c
...
 
 ```shell
 msiexec /i ddagent.msi DDAGENTUSER_NAME=<USERNAME> DDAGENTUSER_PASSWORD=<PASSWORD>
 ```
 
-**Note**: The `<USERNAME>` must be 20 characters or fewer, to comply with Microsoft's [Active Directory Schema (AD Schema) SAM-Account-Name attribute][1].
-
-**Note**: Due to a restriction in the MSI installer, the `DDAGENTUSER_PASSWORD` property cannot contain the semicolon character `;`.
+Requirements:
+* The `<USERNAME>` must be 20 characters or fewer, to comply with Microsoft's [Active Directory Schema (AD Schema) SAM-Account-Name attribute][1].
```
Comment (hestonhoffman) [inline content/en/agent/guide/windows-agent-ddagent-user.md:42]:
```suggestion
* The username must be 20 characters or fewer to comply with Microsoft's [Active Directory Schema (AD Schema) SAM-Account-Name attribute][1].
```

**Review by hestonhoffman — COMMENTED** [review]
Left a couple suggestions

**Pull request merged**